sdk/core/core-amqp/review/core-amqp.api.md

```diff
@@ -22,6 +22,12 @@ import { SenderOptions } from 'rhea-promise';
 import { Session } from 'rhea-promise';
 import { WebSocketImpl } from 'rhea-promise';
 
+// @public
+export interface AcquireOptions {
+    abortSignal?: AbortSignalLike;
+    acquireTimeoutInMs?: number;
```

Comment:
I agree that makes sense. I went back and forth in my head because I didn't want someone to assume it applied also to the task invoked once acquired, but this _should_ only be used by us, and we the docs can make that clear.

Also then it matches AsyncLock.

Reply:
That argument can apply to the abortSignal as well i.e. one could think that it applies to the task. Since the interface is already uses the term "acquire", I agree with Richard that we can skip using the same here.

Reply:
Renamed.